en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -56,20 +56,20 @@ Follow this procedure to test the API revision that you deployed:
 Now you are ready to publish the API to the API Developer Portal so that external consumers can consume the API.
 
 ## Step 4: Publish the API
-Follow this procedure to publish and view the API to the API Developer Portal so that external consumers can consume the API:
+Follow this procedure to publish the API to the API Developer Portal so that external consumers can view and consume the API:
 
 1. Click **Publish** in the left pane.
 2. On the **Lifecycle Management** page, click **Publish**.
-3. Click **Go to Developer Portal**. This signs you in to the API Developer Portal.
+3. Click **Go to Developer Portal**. This signs you into the API Developer Portal.
 
     ![Go to API Developer Portal](../assets/img/apis/go-to-developer-portal.png){.cInlineImage-full}
      
-    You can see the published API in the API Developer Portal.
+    You will see the published API in the API Developer Portal, which looks something like this:
```

Comment:
is this okay to say "something like this?", also using future tense, since you have already logged in to the dev portal?

Reply:
+1 to avoid future tense.
Since the user has signed in to the Developer portal already, it does make sense to avoid future tense here.   
Went with the copy editor’s suggestion on “something like this” since the style guide also suggests to aim for a voice and tone that's conversational [1]. We can decide and remove in case this seems overly colloquial. 
[1] https://developers.google.com/style/tone?hl=en